<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegexReplacementDialog</class>
 <widget class="QDialog" name="RegexReplacementDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>186</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="labelInvalidRegex">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#ff0000;&quot;&gt;Invalid Regular Expression&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelReplacement">
     <property name="text">
      <string>Repace:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelRegex">
     <property name="text">
      <string>Find:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="checkRegularExpression">
       <property name="text">
        <string>Regular Expression</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkIgnoreCase">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Ignore Case</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="textRegex"/>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="textReplacement"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>textRegex</tabstop>
  <tabstop>textReplacement</tabstop>
  <tabstop>checkRegularExpression</tabstop>
  <tabstop>checkIgnoreCase</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RegexReplacementDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
